auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -70,8 +80,52 @@
 
       <!-- Actions -->
       <template v-slot:item-slot-Actions="{ item, index }">
-        <div class="actions mx-auto" :id="`action-menu-${index}`">
-          <span class="open-action">
+        <div class='actions mx-auto' :id='`action-menu-${index}`'>
+          <span class="open-action"  v-if="!!item.affiliationInvites && isCurrentOrganization(item.affiliationInvites[0].fromOrg.id)">
+            <v-btn
+              small
+              color="primary"
+              min-width="5rem"
+              min-height="3rem"
+              max-width="55%"
+              class="open-action-btn"
+              @click="actionHandler(item)"
+            >
+              <span class="text-wrap" v-html="actionButtonText(item)"></span>
+            </v-btn>
+            <!-- More Actions Menu -->
+            <span class="more-actions">
+              <v-menu
+                :attach="`#action-menu-${index}`"
+                v-model="dropdown[index]"
+              >
+                <template v-slot:activator="{ on }">
+                  <v-btn
+                    small
+                    color="primary"
+                    min-height="3rem"
+                    class="more-actions-btn"
+                    v-on="on"
+                  >
+                    <v-icon>{{dropdown[index] ? 'mdi-menu-up' : 'mdi-menu-down'}}</v-icon>
+                  </v-btn>
+                </template>
+                <v-list>
+                  <v-list-item
+                    class="actions-dropdown_item my-1"
+                    @click="openNewAffiliationInvite(item)"
```

Comment:
Is there something feature flagging this?

Reply:
do you think we need to? It shouldn't show up until the backend starts returning the affiliation requests

Reply:
No, is there an example of how it is done, so I can look it up and possibly add things like that.

Reply:
@kialj876 Sounds good, that's the answer I was looking for